### PR TITLE
README update (closing this project)

### LINKED ISSUE #792: Add DEPRECATION warning on Readme and issues template
We need to change the current README with an explanation of the new architecture, link to the new repository, and a notice that this repository is deprecated, unmaintained and is on github only for transparency.

Also add an issue template with this explanation.

### PR DESCRIPTION
**Pull request by andreslucena**
Fixes #792

### CODE REVIEW

**Comment by andreslucena** [comment]
@AjuntamentdeBarcelona/developers we're having problems with CircleCI, I think related with renaming of this repository a few months ago. For me, it isn't important to have this on green status, being a legacy/deprecated version, so we can pass on this one. What do you think?

**Comment by oriolgual** [comment]
I'm OK with it @andreslucena

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+### IMPORTANT NOTE
+
+This is the opensource code repository for "decidim.barcelona", based on [Consul](https://github.com/consul/consul), developed by the Madrid City government. This is a fork that was used between 1 of february of 2016 to the 8 of febreuary of 2017.
```
Comment (oriolgual) [inline ISSUE_TEMPLATE.md]: s/febreuary/february

File: README.md [hunk 2]
```diff
@@ -1,92 +1,21 @@
...
-You'll also need to start `webpack` to serve the needed assets:
+---
 
-```
-webpack -w
-```
+### IMPORTANT NOTE
 
-Prerequisites for testing: install PhantomJS >= 2.0
+This is the opensource code repository for "decidim.barcelona", based on [Consul](https://github.com/consul/consul), developed by the Madrid City government. This is a fork that was used between 1 of february of 2016 to the 8 of febreuary of 2017.
```
Comment (oriolgual) [inline README.md]: s/febreuary/february

**Review by oriolgual — CHANGES_REQUESTED** [review]

**Review by oriolgual — APPROVED** [review]

**Pull request merged**